### DISCUSSION TITLE
Cant see my stats

### DISCUSSION THREAD

**Discussion by bmaksym** [cssbattle/forum#153] [Bugs reports]
When i go to my profile i cant see my stats, i only see this message : Fetching player data, and its loading in the infinity loop, i dont know what to do, any help pls!

**Comment by chinchang** [comment]
Hello @bmaksym Can you please ping us your profile link? We'll check this.

**Reply by siva-kannan3** [reply]
@chinchang same issue happening in my case also. The profile just loading for a long time with the message "fetching player data".
Here's my profile link https://cssbattle.dev/player/siva

**Reply by bmaksym** [reply]
Still i cant see my stats

**Comment by bmaksym** [comment]
https://cssbattle.dev/player/lum1x


Вт, 2 нояб. 2021 г. в 11:37, Kushagra Gour ***@***.***>:

**Comment by smanpreet96** [comment]
I too cannot see my stats. It keeps saying 'fetching player data...'. Here's my profile: https://cssbattle.dev/player/manpreetsingh

**Comment by Maianki** [comment]
Same issue persist with my profile too https://cssbattle.dev/player/cssyoda

**Comment by chinchang** [comment]
The problem is fixed now. If the rank is high, it might take up a lil long to show up, but it will definitely show up now.

**Reply by Maianki** [reply]
> The problem is fixed now. If the rank is high, it might take up a lil long to show up, but it will definitely show up now.

Thanks. I can see my global rank now😊.

**Comment by MagCor** [comment]
I too cannot see my stats. It keeps saying 'fetching player data...'.Help please

**Comment by aammgm** [comment]
Help! I really cant see the stats
my profile: https://cssbattle.dev/player/peach
